PLC4X image building for ARM64 is failing in main

**Describe the bug**

https://dev.azure.com/Edgenesis/shifu/_build/results?buildId=13513&view=logs&j=0112d757-5591-5a73-051b-95347f32b3c6&t=2406ce81-6782-5ee2-6bdb-3e17bb0002ca&l=489 

**To Reproduce**

Steps to reproduce the behavior:

cross build the PLC4x image with `make buildx-push-image-deviceshifu-http-plc4x`

**Expected behavior**

Build success

**Log Output**

```txt

#22 43.24 encoding/xml
#22 43.59 golang.org/x/term
#22 43.86 github.com/mattn/go-isatty
#22 43.93 github.com/mattn/go-colorable
#22 43.99 internal/saferio
#22 44.03 encoding/gob
#22 44.18 github.com/icza/bitio
#22 44.31 github.com/apache/plc4x/plc4go/spi/plcerrors
#22 44.35 github.com/jacobsa/go-serial/serial
#22 44.47 golang.org/x/net/context
#22 ERROR: process "/bin/sh -c CGO_ENABLED=1 GOOS=$TARGETOS GOARCH=$TARGETARCH CGO_LDFLAGS=\"-static\" go build -a -v -o /output/deviceshifu  cmd/deviceshifu/cmdplc4x/main.go" did not complete successfully: exit code: 1
------
 > [linux/amd64->arm64 builder 11/11] RUN CGO_ENABLED=1 GOOS=linux GOARCH=arm64 CGO_LDFLAGS="-static" go build -a -v -o /output/deviceshifu  cmd/deviceshifu/cmdplc4x/main.go:
43.24 encoding/xml
43.59 golang.org/x/term
43.86 github.com/mattn/go-isatty
43.93 github.com/mattn/go-colorable
43.99 internal/saferio
44.03 encoding/gob
44.18 github.com/icza/bitio
44.31 github.com/apache/plc4x/plc4go/spi/plcerrors
44.35 github.com/jacobsa/go-serial/serial
44.47 golang.org/x/net/context
------
Dockerfile.deviceshifuPLC4X:23
--------------------
  21 |     ARG TARGETARCH
  22 |     
  23 | >>> RUN CGO_ENABLED=1 GOOS=$TARGETOS GOARCH=$TARGETARCH CGO_LDFLAGS="-static" go build -a -v -o /output/deviceshifu  cmd/deviceshifu/cmdplc4x/main.go
  24 |     
  25 |     FROM gcr.io/distroless/static-debian11
--------------------
ERROR: failed to solve: process "/bin/sh -c CGO_ENABLED=1 GOOS=$TARGETOS GOARCH=$TARGETARCH CGO_LDFLAGS=\"-static\" go build -a -v -o /output/deviceshifu  cmd/deviceshifu/cmdplc4x/main.go" did not complete successfully: exit code: 1
make: *** [Makefile:62: buildx-push-image-deviceshifu-http-plc4x] Error 1

##[error]Bash exited with code '2'.
Finishing: build deviceshifu-http-plc4x
```

**Server (please complete the following information):**

 - Shifu Version: v0.53.0 with latest
 - Kubernetes distro N/A
 - Version N/A
 - CPU Architecture arm64

**Additional context**

Add any other context about the problem here.

--- a/dockerfiles/Dockerfile.deviceshifuPLC4X
+++ b/dockerfiles/Dockerfile.deviceshifuPLC4X
@@ -1,5 +1,5 @@
 # Build the manager binary
-FROM --platform=$BUILDPLATFORM golang:1.22.0-alpine AS builder
+FROM golang:1.22.0-alpine AS builder
 
 WORKDIR /shifu
 

